improver/cli/sleet_probability.py

```diff
@@ -37,7 +37,9 @@
 @cli.clizefy
 @cli.with_output
 def process(snow: cli.inputcube,
-            rain: cli.inputcube):
+            rain: cli.inputcube,
+            *,
+            ignore_mismatch: bool = False):
```

Comment:
```suggestion
            ignore_mismatch=False):
```
Thi should work with clize fine.

Reply:
Changed.